serpens/sqs.py

```diff
@@ -14,6 +14,47 @@
 
 logger = logging.getLogger(__name__)
 
+SQS_MESSAGE_BATCH_SIZE = 10
+
+
+def send_batch_message(queue_url, entries):
+    client = boto3.client("sqs")
+    response = client.send_message_batch(QueueUrl=queue_url, Entries=entries)
+    if "Failed" in response:
+        failed_ids = [msg["Id"] for msg in response["Failed"]]
+        return failed_ids
+
+
+def publish_message_batch(queue_url, messages, message_attributes={}, batch_size=10):
+    entries = []
+    failed_messages = []
+    if batch_size >= SQS_MESSAGE_BATCH_SIZE:
```

Comment:
Eu não vejo sentido em não permitir um batch size maior que 10. Acho que isso é uma decisão de quem está usando a API.

Reply:
É uma limitação da própria aws: Delivers up to ten messages to the specified queue